client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -1,27 +1,68 @@
 import React from 'react';
+import EachCard from './EachCard.jsx';
 
-class ProductCard extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      placehoder: []
+const ProductCard = (props) => {
+
+
+  let combine = props.detailInfo.reduce((map, value) => {
+    map[value.product_id] = value;
+    return map;
+  }, {});
+  let newArry = [];
+  for (let i = 0; i < props.info.length; i++) {
+    let v2 = combine[props.info[i].id]
```

Comment:
What is v2? I'm thinking maybe "version2"? Is there a better variable name for this?